analysis/topEFT/topeft.py

```diff
@@ -161,7 +161,8 @@ def process(self, events):
         ht = ak.sum(goodJets.pt,axis=-1)
         j0 = goodJets[ak.argmax(goodJets.pt,axis=-1,keepdims=True)]
         #nbtags = ak.num(goodJets[goodJets.btagDeepFlavB > 0.2770])
-        btagwp = 0.4941
+        if year == 2017: btagwp = 0.3040
```

Comment:
We should add a comment here, or maybe in the b-tag README, what working point this is. It looks like maybe `loose`.

Reply:
Those are the cuts for the UL SFs (2017, 2018) of the WP medium, I have added a comment